streaming/index.js

```diff
@@ -187,6 +188,7 @@ const startServer = async () => {
 
   const pgPool = new pg.Pool(pgConfigFromEnv(process.env));
   const server = http.createServer(app);
+  app.use(cors());
```

Comment:
This does look out of place here, but the changed in #27828 reorganises these initial lines, and it becomes placed where it needs to be.